src/Data/Nat/Properties.agda

```diff
@@ -272,8 +272,14 @@ _≥?_ = flip _≤?_
 s≤s-injective : {p q : m ≤ n} → s≤s p ≡ s≤s q → p ≡ q
 s≤s-injective refl = refl
 
+sm≤n⇒m≤pn : suc m ≤ n → m ≤ pred n
```

Comment:
We usually use the full `suc` and `pred` in lemma names, i.e. `suc[m]≤n⇒m≤pred[n]`